Samples/CommandBinding/MauiEdit.WinForms/Program.cs

```diff
@@ -0,0 +1,38 @@
+﻿using MauiEdit.Services;
+using MauiEdit.Services.Extension;
+using MauiEdit.ViewModels;
+
+namespace WinFormsCommandBindingDemo;
+
+static partial class Program
+{
+    /// <summary>
+    ///  The main entry point for the application.
+    /// </summary>
+    [STAThread]
+    static void Main()
+    {
+        Application.SetHighDpiMode(HighDpiMode.SystemAware);
+        Application.EnableVisualStyles();
+        Application.SetCompatibleTextRenderingDefault(false);
+
+        // // Comment this out, to enable the simple WinForms binding demo.
+        // Application.Run(new SimpleTestForm());
+        // return;
+        
+        var mainForm = new MainForm();
+
+        // We need a marshalling control.
```

Comment:
Why do we need a marshalling control?

Reply:
Simplicity, to be honest. To steer around doing the following with the `WindowsFormsSynchronizationContext`:

```
   public async Task<T?> InvokeAsync<T>(Func<T> function)
    {
        TaskCompletionSource<bool> taskCompletionSource = new();
        IAsyncResult? asyncResult = _marshallingControl.BeginInvoke(function);

        var registeredWaitHandle = ThreadPool.RegisterWaitForSingleObject(
               asyncResult.AsyncWaitHandle,
               new WaitOrTimerCallback(InvokeAsyncCallBack),
               taskCompletionSource,
               millisecondsTimeOutInterval: -1,
               executeOnlyOnce: true);

        await taskCompletionSource.Task;

        object? returnObject = _marshallingControl.EndInvoke(asyncResult);
        return (T?)returnObject;
    }
    ```